app/services/region_report_service.rb

```diff
@@ -19,7 +19,7 @@ def initialize(district:, selected_date:, current_user:)
 
   attr_reader :current_user
   attr_reader :data
-  attr_reader :district
+  attr_reader :region
```

Comment:
This change is really nice. Does it mean that, in the interrim, we can create some not-database-backed models like `Block` and `State` to be used by our reporting services? As long as they preserve the necessary interfaces for `name`, `facilities` `registered_patients`, etc?

Reply:
Yeah, we definitely could do that. I imagine that would search on our `block` or `state` text fields and return the related facilities?

Reply:
Following up on our server catchup conversation, yes that would be how we'd do that right away without any data model changes. Our state data is quite clean, so it may be a good next-one to try out. I'm not as confident about the block data.